layouts/_default/single.html

```diff
@@ -22,12 +22,12 @@ <h1 id="page-title"><a href="{{ .Site.BaseURL }}">{{ .Site.Data.config.page_titl
           <li><a href="{{ .Permalink }}">{{ .LinkTitle | humanize }}</a></li>
           {{ end }}
       </ul>
-      {{if $.Site.Data.config.enableToc}}
+      {{if $.Site.Data.config.enableToc}}{{ if not (.Params.disableToc) }}
```

Comment:
Is it possible to combine this with the previous if statement using an `and`? https://gohugo.io/templates/introduction/#conditionals

Reply:
Yes, done! :)